packages/common/infra/src/sync/indexer/impl/indexeddb/__tests__/tokenizer.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from 'vitest';
+
+import { SimpleTokenizer } from '../tokenizer';
+
+test('tokenizer', () => {
+  const tokens = new SimpleTokenizer().tokenize('hello  world,\n AFFiNE');
+
+  expect(tokens).toEqual([
+    { term: 'hello', start: 0, end: 5 },
+    { term: 'world', start: 7, end: 12 },
+    { term: 'affine', start: 15, end: 21 },
```

Comment:
lose the ability of case-sensitive search?

Reply:
The tokenizer is used for full-text search. which generally needs to match as many results as possible, so is case insensitive.

Reply:
what if user want to do case sensitive search

Reply:
I choose to give up this feature.
If you want to support both case-sensitive and case-insensitive simultaneously, you can only store two sets of indexes "AFFiNE" and "affine," which will require twice the storage space.

In a database, if you store the index for "AFFiNE," you won't be able to search for it using "affine."

In Elasticsearch, if case-sensitive search is needed, the indexing method must also be redefined, and it cannot switch to case-insensitive at runtime.